File: server/php/src/Application/Interfaces/SDKClients/ServiceClientInterface.php
<?php

namespace OnlinePayments\ExampleApp\Application\Interfaces\SDKClients;

use OnlinePayments\ExampleApp\Application\DTOs\Service\GetPaymentProductId\RequestDto;
use OnlinePayments\ExampleApp\Application\DTOs\Service\GetPaymentProductId\ResponseDto;

interface ServiceClientInterface
{
    public function getPaymentProductId(RequestDto $requestDto): ResponseDto;
}

File: server/python/app/application/interfaces/sdk_clients/service_client_interface.py
from abc import ABC, abstractmethod
from app.application.dtos.service.get_payment_product_id.request_dto import GetPaymentProductIdRequestDto
from app.application.dtos.service.get_payment_product_id.response_dto import GetPaymentProductIdResponseDto

class ServiceClientInterface(ABC):

    @abstractmethod
    def get_payment_product_id(self, request_dto: GetPaymentProductIdRequestDto) -> GetPaymentProductIdResponseDto:
        pass